.uppercut

```diff
@@ -34,7 +34,8 @@
   <property name="run.metrics" value="false" overwrite="false" />
   <property name="run.nitriq" value="false" overwrite="false" />
   <property name="run.moma" value="false" overwrite="false" />
-  <property name="run.fxcop" value="false" overwrite="false" />
+  <property name="run.fxcop" value="true" overwrite="false" />
+  <property name="run.stylecop" value="true" overwrite="false" />
```

Comment:
Yes. This is actually the file for editing. The defaults file is the one that should not be changed (although the edits you did to it make sense and should go back to the uppercut repo).